Resources/public/templates/translateaction.hbt

```diff
@@ -0,0 +1,29 @@
+<button class="ez-action {{#if disabled}}is-disabled {{else}}{{#if actionId}}action-trigger {{/if}}{{/if}}{{#if hint}}with-hint{{/if}}" data-action="{{ actionId }}">
+    <div class="ez-font-icon action-icon">
+        <p class="action-label">{{ label }}</p>
+        {{#if hint}}
+        <p class="action-hint">{{ hint }}</p>
+        {{/if}}
+    </div>
+</button>
+<div class="ez-expandable-area">
+    <div class="ez-contenttranslations">
+        <h2 class="ez-contenttranslations-title">Existing Translations</h2>
+        <div class="ez-contenttranslations-content">
+            <ul class="ez-contenttranslations-list">
+                {{#each translations}}
+                <li class="ez-contenttranslation">
+                    <a class="ez-contenttranslation-view-link" href="{{ path "viewLocation" id=../location.id languageCode=this }}">
+                        <span class="ez-language-flag flag-icon flag-icon-{{{ getFlagCode this }}}"></span>
+                        {{ this }}
+                    </a>
+                    <a class="ez-contenttranslation-edit-link" href="{{ path "editContent" id=../content.id languageCode=this }}">
+                        Edit
+                    </a>
+                </li>
+                <hr class="ez-separator">
```

Comment:
I have used it to because I haven't got idea how to use only CSS on `<li>` elements to acquire the result projected in wireframes. If I had to render it exactly as it is in wireframes I probably should change `<ul><li>` into divs separated with `<hr>`. If I Had to use `<ul>` for this (as it suppose to be the proper way) then AFAIK I can't render it exactly like it is in wireframes - to problematic is that the separator is not 100% width in wireframes. Which approach should be taken by me?